corehq/apps/app_manager/static/app_manager/js/preview_app.js

```diff
@@ -18,85 +27,108 @@ hqDefine('app_manager/js/preview_app.js', function() {
         OPEN: 'isopen',
         POSITION: 'position',
     };
-    
-    module.initPreviewWindow = function (
-        previewWindowSelector,
-        appManagerBodySelector,
-        previewToggleBtnSelector
-    ) {
-
-        var $appPreview = $(previewWindowSelector);
-        var $appPreviewIframe = $(previewWindowSelector).find('iframe');
-        var $appBody = $(appManagerBodySelector);
-        var $togglePreviewBtn = $(previewToggleBtnSelector);
 
-        $appPreview.data(module.DATA.POSITION, module.POSITION.FIXED);
+    _private.showAppPreview = function() {
+        $(module.SELECTORS.PREVIEW_ACTION_TEXT_SHOW).addClass('hide');
+        $(module.SELECTORS.PREVIEW_ACTION_TEXT_HIDE).removeClass('hide');
+    };
+    _private.hideAppPreview = function() {
+        $(module.SELECTORS.PREVIEW_ACTION_TEXT_SHOW).removeClass('hide');
+        $(module.SELECTORS.PREVIEW_ACTION_TEXT_HIDE).addClass('hide');
+    };
 
-        var _toggleAppPreview = function () {
-            if (localStorage.getItem(module.DATA.OPEN) === module.DATA.OPEN) {
-                localStorage.removeItem(module.DATA.OPEN);
-            } else {
-                localStorage.setItem(module.DATA.OPEN, module.DATA.OPEN);
-            }
-            $(window).trigger(module.EVENTS.RESIZE);
-            if (localStorage.getItem(module.DATA.OPEN)) {
-                _showAppPreview();
-            } else {
-                _hideAppPreview();
-            }
-        };
+    _private.navigateBack = function() {
+        var previewWindow = $appPreviewIframe[0].contentWindow;
```

Comment:
this is a real bug but will be fixed in a later PR (not critical feature that is behind preview flag)